opensrp-anc/src/main/java/org/smartregister/anc/library/util/Utils.java

```diff
@@ -870,27 +869,30 @@ public static String returnTranslatedStringJoinedValue(String value, String key)
                         translatedText = StringUtils.isNotBlank(text) ? NativeFormLangUtils.translateDatabaseString(text, AncLibrary.getInstance().getApplicationContext()) : "";
                         translatedList.add(translatedText);
                     }
-                    return translatedList.size() > 1 ? String.join(",", translatedList) : translatedList.get(0);
+                    return translatedList.size() > 1 ? String.join(",", translatedList) : translatedList.size() == 1 ? translatedList.get(0) : "";
                 } else {
-                    return value;
+                    return value.substring(1, value.length() - 1);
                 }
             }
-            if (value.startsWith("{")) {
+            if (value.charAt(0) == '{') {
```

Comment:
This will break if the value is `null` or `empty` Check that `value` is not blank.